docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -632,9 +632,12 @@ Also note that since wasting and LBWSG are correlated, those with more severe wa
 Assumptions and Limitations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- We are not applying a differential death rate to those effectively covered vs not effectively covered
-- We are generalizing across the whole country. There is likely to be a lot of heterogeneity within the country. 
-- We do not have the durations of untreated SAM and MAM for 0-6 age groups hence we are using the durations from 6-59 age groups. 
+#. We are not applying a differential death rate to those effectively covered vs not effectively covered
+#. We are generalizing across the whole country. There is likely to be a lot of heterogeneity within the country. 
+#. We do not have the durations of untreated SAM and MAM for 0-6 age groups hence we are using the durations from 6-59 age groups. 
+#. We assume that MAM treatment coverage is equal to SAM treatment coverage. Given that SAM treatment is more intensive than MAM treatment, we may underestimate MAM treatment coverage as a result of this assumption.
+#. We assume that MAM and SAM treatment effectivenesses are independent from one another.
+#. We assume that individual simulant's propensity to respond to wasting treatment is not auto-correlated.
```

Comment:
Will this change when we implement the X factor, or will that work by a different mechanism?

Reply:
This will not change with the x-factor. The x-factor only affects the incidence rates to worse wasting categories (which are not affected by treatment), not the recovery rates to better wasting categories (which are affected by treatment)

Reply:
Ok, good to know. Thanks for the clarification.